customer-commons/src/test/resources/features/update_customer.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Update an existing Customer
+
+  Scenario:
+    Given the user wants to update an existing Customer and has valid <Customer UUID> and valid <credentials>
+    When they set the Content-Type header to "application/json"
+    And they set the Accept header to "application/json"
+    And they set the Authentication header to "Bearer <credentials>"
+    And they set the request body to a JSON object describing the Customer
+    And they request PUT /customer/<Customer UUUID>
+    Then they receive a response with status code 200
+    And they see that the response Content-type is "application/json"
+    And they see the response body contains a JSON object describing the Customer
+
+  Scenario:
+    Given the user wants to create a new Customer and has valid <Customer UUID> but has invalid <credentials>
```

Comment:
create?

Reply:
In fact, I don't think the given is necessary

Reply:
But every Scenario should have a Given?

Reply:
Given represents preconditions, if there aren't any I would remove them